analytics/population/nc_updated_pop_comparison.ipynb

```diff
@@ -0,0 +1,991 @@
+{
```

Comment:
<p>It would be good to link to Piero's work.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/hackforla/311-data/pull/1336/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>I have mentioned Piero's name. I do not have a link to his work. He shared the csv file with me. </p>